### PR TITLE
Externalize multiple http/tcp option documentation

### PR DESCRIPTION
**Pull request by mrsaldana**

### CODE REVIEW

**Comment by mrsaldana** [comment]
Fixes #4955

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
Failing tests not related to documentation changes.

File: dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -54,4 +54,10 @@ tcp.hostNameIncludeList=Host name include list
 tcp.hostNameIncludeList.desc=A comma-separated list of host names that are allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid. 
 
 tcp.hostNameExcludeList=Host name exclude list
-tcp.hostNameExcludeList.desc=A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid.
\ No newline at end of file
+tcp.hostNameExcludeList.desc=A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case-sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid, but *.abc.* is not valid.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties]: Suggestion:
A comma-separated list of host names that are not allowed to make inbound connections on this endpoint. Host names are not case sensitive and can start with an asterisk, which is used as a wildcard character. However, asterisks cannot be elsewhere in the host name. For example, *.abc.com is valid; however, *.abc.* is not valid.

File: dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -54,4 +54,10 @@ tcp.hostNameIncludeList=Host name include list
...
+
+tcp.acceptThread=Accept thread
+tcp.acceptThread.desc=If true, then listening ports do not share the same thread for accepting connections. Otherwise, they share the same thread.
+
+tcp.waitToAccept=Wait to accept
+tcp.waitToAccept.desc=Closes connections until the server completely starts up. This is to be used with the acceptThread tcpOption set to true.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties]: Be specific. What closes the connection? If knowing what closes the connections isn't important, then consider writing:
Connections are closed until the server completely starts. Use this property with the acceptThread tcpOption set to true.

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties [hunk 3]
```diff
@@ -95,6 +95,21 @@ http.writeTimeout.desc=Amount of time to wait on a socket for each portion of th
 http.removeServerHeader=Remove server header
 http.removeServerHeader.desc=Removes server implementation information from HTTP headers and also disables the default Liberty profile welcome page.
 
+http.limitNumHeaders=Limit number of headers
+http.limitNumHeaders.desc=Use to limit the number of HTTP headers that can be present in an incoming message.If this limit is exceeded, an error is returned to the remote client.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties]: Suggestion:
Limits the number of HTTP headers that can exist in an incoming message. <add a space> When this limit is exceeded, an error is returned to the remote client.

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties [hunk 4]
```diff
@@ -95,6 +95,21 @@ http.writeTimeout.desc=Amount of time to wait on a socket for each portion of th
...
+
+http.limitFieldSize=Limit header field size
+http.limitFieldSize.desc=Use to enforce the size limits on various HTTP fields, such as request URLs, or individual header names or values. Enforcing the size limits of these fields guards against possible Denial of Service attacks. An error is returned to the remote client if a field exceeds the allowed size.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties]: Suggestion:
Enforces the size limits on various HTTP fields, such as request URLs, or individual header names or values. Enforcing the size limits of these fields guards against possible Denial of Service attacks. An error is returned to the remote client, if a field exceeds the allowed size.

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties [hunk 5]
```diff
@@ -95,6 +95,21 @@ http.writeTimeout.desc=Amount of time to wait on a socket for each portion of th
...
+
+http.doNotAllowDuplicateSetCookies=Do not allow duplicate set cookies
+http.doNotAllowDuplicateSetCookies.desc=Use to prevent the HTTP Channel from sending multiple Set-Cookie headers with the same name.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties]: Suggestion:
Prevents the HTTP Channel from sending multiple Set-Cookie headers with the same name.

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties [hunk 6]
```diff
@@ -95,6 +95,21 @@ http.writeTimeout.desc=Amount of time to wait on a socket for each portion of th
...
+
+http.messageSizeLimit=Message size limit
+http.messageSizeLimit.desc= Used to limit the acceptable size of an incoming message. If a message arrives with a size larger than this value, then an error is returned to the remote client.
```
Comment (bsbyrd1) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties]: Suggestion:
Limits the acceptable size of an incoming message. If a message arrives with a size larger than this value, then an error is returned to the remote client.

File: dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (mrsaldana) [reply dev/com.ibm.ws.channelfw/resources/OSGI-INF/l10n/metatype.properties]: This tcpOption is not being added to the documentation, it already documented. Git just picked up a line break change from the original file.

**Comment by mrsaldana** [comment]
#build

**Comment by mrsaldana** [comment]
Only one failure, which looks to be unrelated to these changes.

**Pull request merged**